modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.tsx

```diff
@@ -49,9 +49,9 @@ const PolicyDetail = ({
 }: ApplicationPolicyDetailsType) => {
 	enum navBarLabels {
```

Comment:
@caiosouza15 can you remove this enum from here?

Move it before `PolicyDetail`

Also, enum best practices recommend to use UpperCase for its name `NavbarLabel`

Reference : https://www.typescriptlang.org/docs/handbook/enums.html